operator/redisfailover/ensurer.go

```diff
@@ -44,6 +44,20 @@ func (w *RedisFailoverHandler) Ensure(rf *redisfailoverv1.RedisFailover, labels
 		if err := w.rfService.EnsureHAProxyDeployment(rf, labels, or); err != nil {
 			return err
 		}
+
+		if rf.Spec.BootstrapNode != nil {
```

Comment:
Why not generate both master and slave haproxy resources when haproxy is enabled?

Reply:
condition removed